apps/easypid/.eas/workflows/e2e-test-ios-with-fingerprint-and-repack.yaml

```diff
@@ -0,0 +1,43 @@
+# https://expo.dev/blog/how-to-integrate-eas-workflows-with-github-actions#fast-e2e-testing-with-fingerprint-and-repack
+# https://docs.expo.dev/eas/workflows/pre-packaged-jobs/#examples-13
+name: Build or repack, then Maestro tests
```

Comment:
Repack is very nice! 

Why do we need both the repack and non-repack file? It seems the repack already automatically builds if there's no build available? Can we remove the other file?